.github/workflows/_build_artifacts.yml

```diff
@@ -60,6 +59,7 @@ jobs:
         with:
           version: ${{ env.VERSION }}
       - name: Delete stale artifacts
+        if: inputs.profile == 'release' && inputs.stage == 'release'
```

Comment:
Would not they be just appended to the list resulting in a huge draft release? Can you please explain why we don't want to delete them?

Reply:
Artifacts are only uploaded on release builds (`main`) -- if I don't add this, then PRs will remove the drafted assets

Reply:
This job would then just update the body content and labels on each PR